baxter_collaboration_lib/include/robot_utils/ros_thread_image.h

```diff
@@ -40,6 +40,7 @@ class ROSThreadImage
     cv::Mat  curr_img;
     cv::Size img_size;
     bool    img_empty;
+    std::string _encoding;
```

Comment:
as per convention, data members should NOT have `_` before. Also , fix alignment. It would be nice also to add a one-line comment for all of the members. In general, improve what you can improve when you touch a codebase!